heartbeat/tracer/tracer_test.go

```diff
@@ -123,7 +123,8 @@ func listenTilClosed(t *testing.T, sockPath string) []string {
 
 	conn, err := listener.Accept()
 	require.NoError(t, err)
-	var received []string
+	// no need to pre-allocate, but it seems to make the linter happy
+	received := make([]string, 0, 10)
```

Comment:
Nit: you could omit the capacity here.
```suggestion
	received := make([]string, 0)
```

Reply:
Sure, but IIRC the linter wanted it in there, and it doesn't hurt.

Reply:
I think the linter was complaining more about the `nil` slice initialization.